### ISSUE TITLE
Final Peer Review_my463 

### ISSUE THREAD

**Issue by PatrickYuanMingcan** [Archerhaha/LOLGameWinPrediction#14]
The whole project is about to predict the result of LOL games using the features such as first blood and first tower, and analyze the relationship between the features and result to find the most relevant feature to increase the performance for the video game team.

I think it is a very interesting topic because the market of video game is booming and its vast betting market could help us make a great profit. It is the economic value of this topic. Also, the data of the video game is easy to collect and there is no missing data problem because all of the data would be recorded. So the data would be clean.For the analysis, it seems you guys use different loss functions and regularizations to improve the prediction. And the result of misclassification shows that the methods work well.

However, I still have some suggestions for you to improve the prediction. Firstly, you guys use k-fold cross validation to select the model with best performance, while you just split data into training set and test set and use test set error as misclassification rate. It is obvious that you need to create a validation set to select model and test set to measure the errors.

And for champion selection part, I think the selection is more related to the version of the game, which mean this rank need be updated with the game. So the rank can't be used to predict the result of game. For different misclassification rate of different model, could you guys explain the results more specifically? Why the random forest has the best rate?

To sum up, you guys did a great job because I'm also very interesting in betting market of video games, and it seems the result is well enough to apply in real market.
